src/feature/hs/hs_client.c

```diff
@@ -1261,12 +1471,12 @@ hs_client_note_connection_attempt_succeeded(const edge_connection_t *conn)
  * newly allocated descriptor object.
  *
  * Return 0 on success else a negative value and desc is set to NULL. */
-int
+hs_desc_decode_status_t
 hs_client_decode_descriptor(const char *desc_str,
                             const ed25519_public_key_t *service_identity_pk,
                             hs_descriptor_t **desc)
 {
-  int ret;
+  hs_desc_decode_status_t ret;
```

Comment:
Please initialize to something for future proofing.

Reply:
See my reasoning above.